ImprovedAnvil/lib/spells/03_patches_vanilla.tph

```diff
@@ -75,6 +75,172 @@ COPY_EXISTING ~SPIN999.spl~ ~override/SPIN999.spl~
+    LAUNCH_PATCH_FUNCTION ~ALTER_SPELL_EFFECT~ 
+      INT_VAR
+	    header = %ability_num%
+        match_opcode = 111
+      STR_VAR
+	    resource = ~S!seek01~
+	END
+  END  
+  PATCH_FOR_EACH ~ability_num~ IN "10" "11" BEGIN
+    LAUNCH_PATCH_FUNCTION ~ALTER_SPELL_EFFECT~ 
+      INT_VAR
+	    header = %ability_num%
+        match_opcode = 111
+      STR_VAR
+	    resource = ~S!seek02~
+	END
+  END  
+  PATCH_FOR_EACH ~ability_num~ IN "12" "13" "14" BEGIN
+    LAUNCH_PATCH_FUNCTION ~ALTER_SPELL_EFFECT~ 
+      INT_VAR
+	    header = %ability_num%
+        match_opcode = 111
+      STR_VAR
+	    resource = ~S!seek03~
+	END
+  END 
+  PATCH_FOR_EACH ~ability_num~ IN "15" "16" "17" "18" "19" BEGIN
+    LAUNCH_PATCH_FUNCTION ~ALTER_SPELL_EFFECT~ 
+      INT_VAR
+	    header = %ability_num%
+        match_opcode = 111
+      STR_VAR
+	    resource = ~S!seek04~
+	END
+  END 
+  LAUNCH_PATCH_FUNCTION ~ALTER_SPELL_EFFECT~ 
+    INT_VAR
+	  header = 20
+      match_opcode = 111
+    STR_VAR
+	  resource = ~S!seek05~
+  END
+  LPF ~S!CLONE_SPELL_HEADER~
+    INT_VAR
+      header_to_copy = 19
+      level_req = 25
+  END
+  LAUNCH_PATCH_FUNCTION ~ALTER_SPELL_EFFECT~ 
+    INT_VAR
+	  header = 21
+      match_opcode = 111
+    STR_VAR
+	  resource = ~S!seek06~
+  END
+  // Make the spell undispellable
+  LPF ~ALTER_SPELL_EFFECT~
+    INT_VAR
+      match_opcode = ~-1~
+	  resist_dispel = 0
+  END
+
+// Boon of Lathander
+COPY_EXISTING ~SPCL741.spl~ ~override/SPCL741.spl~
+  SAY UNIDENTIFIED_DESC @12120
+  // Give additional +1 thaco/dmg from lvl 11 up
+  PATCH_FOR_EACH ~ability_num~ IN "2" "3" "4" "5" "6" "7" "8" "9" "10" "11" BEGIN
+    LAUNCH_PATCH_FUNCTION ~ALTER_SPELL_EFFECT~ 
+      INT_VAR
+	    header = %ability_num%
+        match_opcode = 73
+		parameter1 = 2
+	END
+    LAUNCH_PATCH_FUNCTION ~ALTER_SPELL_EFFECT~ 
+      INT_VAR
+	    header = %ability_num%
+        match_opcode = 54
+		parameter1 = 2
+	END
+  END  
+
+// Weaken Undead
+COPY_EXISTING ~SPCL742.spl~ ~override/SPCL742.spl~
+  SAY UNIDENTIFIED_DESC @12089
+  // remove all but one spell abilities
+  PATCH_FOR_EACH ~min_lvl_val~ IN "2" "3" "4" "5" "6" "7" "8" "9" "10" "11" "12" "13" "14" "15" "16" "17" "18" "19" BEGIN
+    LAUNCH_PATCH_FUNCTION ~DELETE_SPELL_HEADER~ 
+      INT_VAR
+	    header_type = ~-1~
+        min_level = ~%min_lvl_val%~
+	END
+  END
+  // change projectile to None, to make it single target
+  LPF ~ALTER_SPELL_HEADER~
+    INT_VAR
+      projectile  = 1
+  END
+  // remove Hold EFF
+  LPF ~DELETE_SPELL_EFFECT~
+    INT_VAR
+      opcode_to_delete = 185
+  END
+  // make all initial effects bypass MR and allow no saving throw
+  LPF ~ALTER_SPELL_EFFECT~
+    INT_VAR
+      check_globals = 0
+      resist_dispel = 3
+	  savingthrow = 0
+  END
+  // Add effects slowing movements and reducing physical resistances with no saving throw
+  PATCH_FOR_EACH ~eff_name~ IN "S!SLD-10" "S!CRD-10" "S!SLD-10" "S!CRD-10" "S!MVM50" BEGIN
```

Comment:
formating and tabs all over the place here, better to maintain consistency

Reply:
I think that Notepad++ uses tabs when I add a new line that is indented, will have to change that, sorry.

Reply:
Ok, I fixed most of that

Reply:
@roman-d  if you think that any of those issues is resolved, then please close the comment thread, so I know it's ok now